satpy/readers/ahi_hsd.py

```diff
@@ -494,8 +517,19 @@ def calibrate(self, data, calibration):
     def convert_to_radiance(self, data):
         """Calibrate to radiance."""
 
-        gain = self._header["block5"]["gain_count2rad_conversion"][0]
-        offset = self._header["block5"]["offset_count2rad_conversion"][0]
+        bnum = self._header["block5"]['band_number'][0]
+        # Check calibration mode and select corresponding coefficients
+        if (self.calib_mode == "UPDATE" and bnum < 7):
```

Comment:
Parentheses are not needed here or in the below `if`.

Reply:
Fixed in new commit